### PR TITLE
add draft for assistant-rights

### PR DESCRIPTION
**Pull request by hiaux0**

### CODE REVIEW

**Comment by hiaux0** [comment]
@JaclyNolan 

what other rights do you want? :)))

**Comment by JaclyNolan** [comment]
> @JaclyNolan
> 
> what other rights do you want? :)))

* 100k (dollar) bonus each year
* health/insurance benefits
* "bi-a" table in HQ

File: knowledge-graph/Assistant Working Guidelines/Assistant Rights.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+- Also, you are always free to share your thoughts, and any other issue/topic with me!
+- I may not immediately answer to those, in which case you can create a ticket
```
Comment (JaclyNolan) [inline knowledge-graph/Assistant Working Guidelines/Assistant Rights.md:2]: Create a "ticket" definition in Definition.md pls

Comment (hiaux0) [reply knowledge-graph/Assistant Working Guidelines/Assistant Rights.md:2]: done

**Review by JaclyNolan — APPROVED** [review]
Acknowledged #Ticket

**Pull request merged**